db/schema.rb

```diff
@@ -1,55 +1,61 @@
-    t.index ["user_id"], name: "index_comments_on_user_id"
-  end
-
-  create_table "likes", force: :cascade do |t|
-    t.bigint "author_id", null: false
-    t.datetime "created_at", null: false
-    t.datetime "updated_at", null: false
-    t.index ["author_id"], name: "index_likes_on_author_id"
-  end
-
-  create_table "posts", force: :cascade do |t|
-    t.bigint "author_id", null: false
-    t.string "title"
-    t.text "text"
-    t.integer "comments_counter"
-    t.integer "likes_counter"
-    t.datetime "created_at", null: false
-    t.datetime "updated_at", null: false
-    t.index ["author_id"], name: "index_posts_on_author_id"
-  end
-
-  create_table "users", force: :cascade do |t|
-    t.string "name"
-    t.string "photo"
-    t.text "bio"
-    t.integer "posts_counter"
-    t.datetime "created_at", null: false
-    t.datetime "updated_at", null: false
-  end
-
-  add_foreign_key "comments", "users"
-  add_foreign_key "likes", "users", column: "author_id"
-  add_foreign_key "posts", "users", column: "author_id"
-end
+# This file is auto-generated from the current state of the database. Instead
+# of editing this file, please use the migrations feature of Active Record to
+# incrementally modify your database, and then regenerate this schema definition.
+#
+# This file is the source Rails uses to define your schema when running `bin/rails
+# db:schema:load`. When creating a new database, `bin/rails db:schema:load` tends to
+# be faster and is potentially less error prone than running all of your
+# migrations from scratch. Old migrations may fail to apply correctly if those
+# migrations use external dependencies or application code.
+#
+# It's strongly recommended that you check this file into your version control system.
+
+ActiveRecord::Schema[7.0].define(version: 2023_08_24_164455) do
+  # These are extensions that must be enabled in order to support this database
+  enable_extension "plpgsql"
+
+  create_table "comments", force: :cascade do |t|
+    t.bigint "author_id", null: false
+    t.bigint "post_id", null: false
+    t.text "text"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.index ["author_id"], name: "index_comments_on_author_id"
+    t.index ["post_id"], name: "index_comments_on_post_id"
+  end
+
+  create_table "likes", force: :cascade do |t|
+    t.bigint "author_id", null: false
+    t.bigint "post_id", null: false
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.index ["author_id"], name: "index_likes_on_author_id"
+    t.index ["post_id"], name: "index_likes_on_post_id"
+  end
+
+  create_table "posts", force: :cascade do |t|
+    t.bigint "author_id", null: false
+    t.string "title"
+    t.text "text"
+    t.integer "comments_counter"
+    t.integer "likes_counter"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.index ["author_id"], name: "index_posts_on_author_id"
+  end

```

Comment:
- [ ] Whenever I've tried to create a Post , a Comment or a Like , the database is not saving data. It's ROLINGBACK, Kindly make sure that data are saved to the database 👍 

**Post Model**
![Capture](https://github.com/uno36/blog-app-rails/assets/80458199/127fbd3b-21ed-4f92-85b2-53a97f07a47a)

**Like Model**
![image](https://github.com/uno36/blog-app-rails/assets/80458199/479bc6f7-a7c4-4ac0-80d4-60710777be24)

**Comment Model**
![Capture1](https://github.com/uno36/blog-app-rails/assets/80458199/3a89c8ae-1411-4171-8e2c-74b0661b4916)